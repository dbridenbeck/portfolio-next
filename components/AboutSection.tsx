import styled from "styled-components";
import Pill from "./SharedComponents/Pill";
import { devices } from "../utils/cssBreakpoints";

const AboutContainer = styled.div`
  display: flex;
  flex-direction: column;
  justify-content: flex-start;
  height: 100%;
`;

const Headline = styled.h2`
  font-size: 1.75em;
  font-weight: 600;
  @media ${devices.mobileLandscape} {
    margin: 0.125em 0;
    font-size: 2.25em;
  }
`;

const Info = styled.p`
  font-size: 0.825em;
  @media ${devices.mobileLandscape} {
    margin: 0.75em 0;
    font-size: 0.85em;
  }
`;

const SkillsSection: React.FC = () => (
  <AboutContainer>
    <Headline>
      My background is in life coaching, solutions engineering, vendor
      management, and sales.
    </Headline>
    <Info>
      When I’m not working, I’m hanging out with my wife and daughter, playing
      pinball, or rollerskating.
    </Info>
    <Info>
      You can also find me volunteering for the{" "}
      <a href="https://github.com/codeforpdx/dwellingly-app" target="_blank">
        Dwelling.ly
      </a>{" "}
      project at Code for PDX and participating in coding meetups like Portland
      ReactJS and Mentorship Saturdays.
    </Info>
  </AboutContainer>
);

export default SkillsSection;
