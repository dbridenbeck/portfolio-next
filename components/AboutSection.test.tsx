import React from "react";
import { render } from "@testing-library/react";
import AboutSection from "./AboutSection";

describe("AboutSection", () => {
  it("should render without errors", () => {
    const { container } = render(<AboutSection />);
    expect(container).toMatchSnapshot();
  });
});
