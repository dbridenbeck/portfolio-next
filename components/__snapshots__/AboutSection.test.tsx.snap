// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutSection should render without errors 1`] = `
<div>
  <div
    class="AboutSection__AboutContainer-sc-1rrnv9m-0 dAjSqw"
  >
    <h2
      class="AboutSection__Headline-sc-1rrnv9m-1 epRgFa"
    >
      My background is in life coaching, solutions engineering, vendor management, and sales.
    </h2>
    <p
      class="AboutSection__Info-sc-1rrnv9m-2 fbwaey"
    >
      When I’m not working, I’m hanging out with my wife and daughter, playing pinball, or rollerskating.
    </p>
    <p
      class="AboutSection__Info-sc-1rrnv9m-2 fbwaey"
    >
      You can also find me volunteering for the
       
      <a
        href="https://github.com/codeforpdx/dwellingly-app"
        target="_blank"
      >
        Dwelling.ly
      </a>
       
      project at Code for PDX and participating in coding meetups like Portland ReactJS and Mentorship Saturdays.
    </p>
  </div>
</div>
`;
